my-videos/src/App.js

```diff
@@ -1,11 +1,10 @@
-import logo from './logo.svg';
 import MyVideos from './components/MyVideos/index';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <MyVideos />
+      <MyVideos projectsUrl={'./projects-sample-response.json'}/>
```

Comment:
Would you mind double-checking this path? 

```suggestion
      <MyVideos projectsUrl={'../public/projects-sample-response.json'}/>
```

or one step further  -- could you move the path to a constant, and leverage react router here?